docs/orchestration/index.md

```diff
@@ -56,68 +52,39 @@ applied is: `change detection`, `current directory`, `wants`.
-the fields **before** and **after**.
+Within Terramate, we can arrange stacks in a filesystem hierarchy. Parent stacks are always executed before their child stacks in this arrangement. Thus, if **stack A** includes **stack B**, **stack A** will always be executed first.
 
-Each field is a set of string (**set(string)**), where each string is a path that
-references another directory, which can be a stack or contain stacks inside.
+This process carries both benefits and risks. On the positive side, it offers you the ability to rearrange stacks, which can improve the mirroring of your cloud infrastructure and control the sequence of execution, all without changing any code. On the flip side, you must exercise caution. Rearranging stacks merely for aesthetic appeal might unintentionally change the execution sequence, which could lead to operational problems.
 
-The explicit order can be used in conjunction with the implicit filesystem order
-but have in mind that a parent stack in the filesystem can never execute after a
-child one, and trying to make this using explicit **before** and **after** clauses
-will lead to a cycle error.
 
-References to directories can be relative to the stack being configured in the form:
+### Explicit Order Of Execution
 
-```
-../../dir
-```
+Terramate's Explicit Order of Execution feature allows you to designate a specific order for stack execution. This is done by defining **before** and **after** fields within the **stack** block, referring to other directories containing stacks.
 
-Or they can be relative to the project root, starting with "/":
+**Before** ensures that the specified stack is executed before all stacks in the mentioned directories. Conversely, **after** ensures the stack is executed after those in the directories. Both relative and project root paths can be used to reference directories.
 
-```
-/path/relative/to/project/root/dir
-```
+> **Note:** A parent stack can never be executed after a child stack, as this would cause a cycle error.
 
-**before** ensures that the configured stack is executed before the
-stacks contained in the directory paths. As the stack you are saying:
-"I execute before all stacks inside these directories".
+Here's how to use this feature:
 
-**after** ensures the opposite, that the stacks contained in the provided
-directories are executed before the current stack. As the stack, you are saying:
-"I execute after all stacks inside these directories".
+1. Choose the stack directory you want to change it's order of execution
+2. Open your Terramate configuration file `stack.tm.hcl` in a text editor.
+2. Select the stack block you want to allocate a specific order.
+3. Depending on your needs, add either the **before** or **after** field inside the stack block.
+4. Add **string** values representing the directory paths you want to order execution for in the **before** or **after** field.
+5. Save and close the file.
```

Comment:
Just my opinion, feel free to ignore.

This kind of explanation stands out from the rest of the documentation. For all other configurations, we never say "open the file, add the field, save and close the file" ...
This is a configuration language after all, so users can understand if we just say:

```
This can be configured by adding `before` and `after` fields to the stack block. Something like:

stack {
    before = [
        "/other/stack"
    ]
}
```